client/src/features/auth/authAPI.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const API_URL = `${process.env.REACT_APP_BASE_API_URL}/users`;
+
+export const login = (credentials) =>
+  axios.post(`${API_URL}/login`, credentials);
+
+export const register = (userData) =>
+  axios.post(`${API_URL}/register`, userData);
+
+export const getCurrentUser = () => {
+  const accessToken = localStorage.getItem("accessToken");
+  return axios.get(`${API_URL}/current`, {
+    headers: {
+      Authorization: accessToken,
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider adding Bearer prefix to token</b></div>
<div id="fix">

Consider adding the `Bearer` prefix to the Authorization header value. Most authentication systems expect the token in the format `Bearer <token>`.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
      Authorization: `Bearer ${accessToken}`,
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Divyanshu9822/blog-mern-app/pull/22#issuecomment-2544587479>#245782</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged